src/testing/complete-redux-store-dump.js

```diff
@@ -0,0 +1,2811 @@
+              'local-monitors': '',
+              master: '',
+              monitors: '',
+              munin: '',
+              'nrpe-external-master': '',
+              'shared-db': '',
+              slave: ''
+            },
+            publicAddress: ''
+          }
+        },
+        remoteApplications: {},
+        offers: {},
+        relations: [
+          {
+            id: 0,
+            key: 'haproxy:peer',
+            'interface': 'haproxy-peer',
+            scope: 'global',
+            endpoints: [
+              {
+                application: 'haproxy',
+                name: 'peer',
+                role: 'peer',
+                subordinate: false
+              }
+            ],
+            status: {
+              status: 'joined',
+              info: '',
+              data: {},
+              since: '2019-08-20T03:58:06.931395952Z',
+              kind: '',
+              version: '',
+              life: ''
+            }
+          },
+          {
+            id: 5,
+            key: 'mysql:cluster',
+            'interface': 'mysql-ha',
+            scope: 'global',
+            endpoints: [
+              {
+                application: 'mysql',
+                name: 'cluster',
+                role: 'peer',
+                subordinate: false
+              }
+            ],
+            status: {
+              status: 'joined',
+              info: '',
+              data: {},
+              since: '2019-08-20T03:58:06.78747013Z',
+              kind: '',
+              version: '',
+              life: ''
+            }
+          }
+        ],
+        controllerTimestamp: '2019-08-29T19:46:26.742014007Z'
+      },
+      '4e8a5223-c34a-4ac9-83ca-1175165467d7': {
+        model: {
+          name: 'frontend-ci',
+          type: 'iaas',
+          cloudTag: 'cloud-google',
+          region: 'us-east1',
+          version: '2.4.7',
+          availableVersion: '',
+          modelStatus: {
+            status: 'suspended',
+            info: 'suspended since cloud credential is not valid',
+            data: {},
+            since: null,
+            kind: '',
+            version: '',
+            life: ''
+          },
+          meterStatus: {
+            color: '',
+            message: ''
+          },
+          sla: 'unsupported'
+        },
+        machines: {
+          '2': {
+            agentStatus: {
+              status: 'down',
+              info: 'agent is not communicating with the server',
+              data: {},
+              since: '2019-01-08T00:54:52.357321928Z',
+              kind: '',
+              version: '2.4.7',
+              life: ''
+            },
+            instanceStatus: {
+              status: 'running',
+              info: 'RUNNING',
+              data: {},
+              since: '2016-12-12T17:36:03.774497893Z',
+              kind: '',
+              version: '',
+              life: ''
+            },
+            dnsName: '104.196.205.152',
```

Comment:
FWIW this seems to resolve but throws a 404 🤷‍♀

Reply:
Hah, yeah I just randomly typed in numbers for the uuid's and ip's. I suppose there is always bound to be something real on the other end (especially for ipv4)